src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+is_a: MONDO:0005583 {source="OMIA:000105"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0005298 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog osteoporosis
+
+[Term]
+id: MONDO:1010296
+name: Marfan syndrome, non-human animal
+def: "Marfan syndrome that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000628 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000628"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0007947 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog Marfan syndrome
+
+[Term]
+id: MONDO:1010297
+name: osteoarthritis, non-human animal
+def: "Osteoarthritis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000747 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000747"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0005178 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog osteoarthritis
+
+[Term]
+id: MONDO:1010298
+name: osteochondritis dissecans, non-human animal
+def: "Osteochondritis dissecans that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000748 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000748"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0017178 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog osteochondritis dissecans
+
+[Term]
+id: MONDO:1010299
+name: osteochondrosis, non-human animal
+def: "Osteochondrosis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000750 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000750"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0018381 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog osteochondrosis
+
+[Term]
+id: MONDO:1010300
+name: osteopetrosis, non-human animal
+def: "Osteopetrosis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000755 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000755"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0017198 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog osteopetrosis
+
+[Term]
+id: MONDO:1010301
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.